lib/resource/Tenant.js

```diff
@@ -220,4 +220,21 @@ Tenant.prototype.getCustomData = function getCustomData(/* [options,] callback *
   return this.dataStore.getResource(this.customData.href, args.options, require('./CustomData'), args.callback);
 };
 
+/**
+ * Retrieves all the {@link IdSiteModel} resources for this resource.
+ *
+ * @param {CollectionQueryOptions} [collectionQueryOptions]
+ * Options for querying, paginating, and expanding the collection.
+ *
+ * @param {Function} callback
+ * The function to call when then the operation is complete. Will be called
+ * with the parameters (err, {@link CollectionResource}). The collection will
+ * be a list of {@link IdSiteModel} objects.
+ *
+ */
+Tenant.prototype.getIdSiteModels = function getTenantIdSiteModels(/* [options,] callback */) {
```

Comment:
Regarding this, I was following a comment in the IdSiteModel class file which references `{@link Tenant#getIdSiteModels Tenant.getIdSiteModels()}` (which did not exist at the time). Updating this as well.